packages/storefront-lwc/package.json

```diff
@@ -14,42 +14,45 @@
-    "build:development": "lwc-services build --noclear",
+    "start": "cross-env NODE_ENV=production node -r esm ./scripts/runtime.js",
+    "start:dev": "run-p watch:client watch:server watch:scss",
+    "clean:stats": "rm -rf dist/stats.json && rm -rf dist/report.html",
+    "build": "yarn build:lwc:scss && yarn run build-lwc && yarn build:public:scss",
     "lint": "eslint ./src/**/*.js",
     "prettier": "prettier --write \"**/*.{css,html,js,json,md,ts,yaml,yml}\"",
     "prettier:verify": "prettier --list-different \"**/*.{css,html,js,json,md,ts,yaml,yml}\"",
-    "start": "cross-env NODE_ENV=production NODE_OPTIONS=\"--experimental-modules --experimental-specifier-resolution=node\" node ./scripts/runtime.mjs",
-    "test:unit": "lwc-services test:unit",
-    "test:unit:coverage": "lwc-services test:unit --coverage",
-    "test:unit:debug": "lwc-services test:unit --debug",
-    "test:unit:watch": "lwc-services test:unit --watch",
-    "start:dev": "run-p watch:client watch:server watch:scss",
-    "watch:client": "lwc-services watch",
+    "test:unit": "jest",
+    "test:unit:coverage": "jest --coverage",
+    "test:unit:debug": "jest --debug",
+    "test:unit:watch": "jest --watch",
+    "watch:client": "rollup --config ./scripts/rollup.config.js --watch",
     "watch:server": "nodemon",
-    "build:scss": "yarn build:public:scss && yarn build:lwc:scss",
+    "watch:scss": "run-p -c --aggregate-output watch:lwc:scss watch:public:scss",
     "build:lwc:scss": "node-sass src/modules/commerce -o src/modules/commerce --include-path node_modules",
-    "build:public:scss": "node-sass src/scss -o dist/public/assets/css --include-path node_modules",
-    "watch:scss": "yarn watch:lwc:scss",
-    "watch:lwc:scss": "yarn build:lwc:scss && node-sass src/modules/commerce -wo src/modules/commerce --include-path node_modules",
-    "watch:public:scss": "yarn build:public:scss && node-sass src/scss -wo dist/public/assets/css --include-path node_modules"
+    "watch:lwc:scss": "npm run build:lwc:scss && node-sass src/modules/commerce -wo src/modules/commerce --include-path node_modules",
+    "build:public:scss": "node-sass src/scss -o dist/css --include-path node_modules",
+    "watch:public:scss": "yarn run build:public:scss && node-sass src/scss -wo dist/css --include-path node_modules",
+    "build-lwc": "cross-env NODE_ENV=production rollup -c scripts/rollup.config.js"
   },
   "jest": {
     "preset": "@lwc/jest-preset",
     "moduleNameMapper": {
       "^commerce/(.+)": "<rootDir>/src/modules/commerce/$1/$1"
-    }
+    },
+    "collectCoverageFrom": [
+      "src/**/*.{js,ts}",
+      "app/**/*.{js,ts}",
+      "!app/api.example.js"
+    ]
   },
   "dependencies": {
-    "@lwc/synthetic-shadow": "^1.1.1",
+    "@lwc/synthetic-shadow": "1.4.0-alpha3",
     "@lwce/apollo-client": "^0.0.1",
+    "@lwce/router": "^0.1.1",
```

Comment:
@blittle will be using this for routing instead of universal router?

Reply:
Also will this need 3PP approval? dont know how that works for packages created within.

Reply:
This isn't supposed to be a part of this PR, but a next PR. And yes, it will need approval.

Reply:
(I'll pull it out)

Reply:
yeah i saw it is not implemented, but it does look great (looked at the documentation). I am fine whether you leave it here or remove it for now, i dont care. approving pr.